src/customize-pop/workspace-settings.md

```diff
@@ -0,0 +1,45 @@
+# Workspaces Settings
+
+Workspace placement and behavior is controlled in the `Workspaces` tab. Learn more about using Workspaces in the [Using Workspaces](/navigate-pop/using-workspaces.md) section.
+
+![Workspace Settings](/images/customize-desktop/workspace-settings.png)
+
+## Dynamic Workspaces
+
+Automatically remove workspaces when they contain no active windows, and automatically generate new workspaces when a window is dragged to the next lowest workspace.
+
+<video autoplay loop>
+    <source src="/images/customize-desktop/dynamic-workspaces.webm" />
```

Comment:
This is a really good loop! It would be a _lot_ more satisfying if it didn't cut away before selecting the first desktop again at the end (and if the windows ended up in the same positions at the end as they started out in.)

Reply:
Fixed.